client/app/components/app-header/components/FavoritesDropdown.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useMemo, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty, template } from 'lodash';
+
+import Dropdown from 'antd/lib/dropdown';
+import Icon from 'antd/lib/icon';
+import Menu from 'antd/lib/menu';
+
+import { HelpTrigger } from '@/components/HelpTrigger';
+
+export default function FavoritesDropdown({ fetch, urlTemplate }) {
+  const [items, setItems] = useState();
+  const [loading, setLoading] = useState(false);
+
+  const noItems = isEmpty(items);
+  const urlCompiled = useMemo(() => template(urlTemplate), []);
+
+  const fetchItems = useCallback(() => {
+    setLoading(true);
+    fetch().$promise.then(({ results }) => {
+      setLoading(false);
+      setItems(results);
+    });
```

Comment:
It worths using `.finally(() => setLoading(false))`, to avoid breaking in this case:

![image](https://user-images.githubusercontent.com/3356951/66845903-491f1100-ef47-11e9-8654-c2fffec1a5e4.png)

Reply:
Very true. Fixed!